log4j-core/src/test/java/org/apache/logging/log4j/core/HostNameTest.java

```diff
@@ -16,21 +16,24 @@
  */
 package org.apache.logging.log4j.core;
 
+import static org.hamcrest.Matchers.endsWith;
+import static org.hamcrest.Matchers.hasSize;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertThat;
+
```

Comment:
static imports go last. We also allow wildcard imports of static imports so this change isn't required.